src/api/app/views/webui2/webui/packages/job_history/index.html.haml

```diff
@@ -0,0 +1,48 @@
+- @pagetitle = "Job history of #{@project} / #{@package_name}"
+- @metarobots = 'index,nofollow'
+
+.card
+  = render partial: 'webui/package/tabs', locals: { project: @project, package: @package }
+
+  .card-body
+    %h3= @pagetitle
+    %h6.subtitle
+      Repository / Architecture: #{params[:repository]} / #{params[:arch]}
+
+    %table.responsive.table.table-striped.table-bordered#jobhistory-table{ width: '100%' }
```

Comment:
I believe `{ width: '100%' }` could be replaced by [.w-100](https://getbootstrap.com/docs/4.1/utilities/sizing/)